triage/Classes/AppDelegate.swift

```diff
@@ -11,55 +11,40 @@ import UIKit
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
+  let API = ZendeskAPI.instance
+
   var window: UIWindow?
   var storyboard = UIStoryboard(name: "Main", bundle: nil)
 
-
   func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
-    NSNotificationCenter.defaultCenter().addObserver(self, selector: "userDidLogout", name: "userDidLogoutNotification", object: nil)
-    if User.currentUser != nil {
-      if User.currentUser?.name == nil {
-        User.currentUser = nil // something messed up with the stored user
-      } else {
-        var vc = storyboard.instantiateViewControllerWithIdentifier("TicketsViewController") as TicketsViewController
-        vc.tickets = Tickets(fixture: "tickets")
-        window?.rootViewController = vc
-      }
```

Comment:
We should be storing and checking credentials as opposed to `currentUser`. The PR goes as far to accomplish that. In addition, rather than managing the access token off the user, it's stored for the client (AFOAuth2Manager has an API for doing this succinctly).